Makefile

```diff
@@ -6,7 +6,7 @@ SRCDIR           ?= .
 BUILD_DIR        ?= $(CURDIR)/bin
 COVERAGE_FILE    ?= coverage.out
 
-GO_VERSION       ?= 1.22
+GO_VERSION       ?= 1.22.8
```

Comment:
until renovate an automations are in place i would just pinned the major so we get the patch on every release.

Reply:
Needed the patch to download and install in the Dockerfile, but changed it to fetch the latest go patch version based on `major.minor`. Can be simplified once we have renovate. Thanks for pointing it out.